docs/_includes/browsers.md

```diff
@@ -3,7 +3,7 @@ that you use the most up-to-date browser that's compatible with your operating s
 supported.
 
 * The new [Microsoft Edge](https://www.microsoft.com/edge)
-* Safari (latest version, Mac only)
+* [Safari](https://www.apple.com/safari) (latest version, Mac only)
```

Comment:
```suggestion
* [Safari](https://www.apple.com/safari) (latest version)
```

We can also remove "Mac only" since we are supported on mobile as well :)